distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -30,6 +30,24 @@ javax.net.ssl.trustStore=etc/keystores/serverTruststore.jks
 javax.net.ssl.trustStorePassword=changeit
 javax.net.ssl.keyStoreType=jks
 
+# Set the global url properties
+org.codice.ddf.system.protocol=https://
+org.codice.ddf.system.externalHostname=localhost
+org.codice.ddf.system.externalHttpsPort=8993
+org.codice.ddf.system.externalHttpPort=8181
+org.codice.ddf.system.hostname=localhost
+org.codice.ddf.system.httpsPort=8993
+org.codice.ddf.system.httpPort=8181
+org.codice.ddf.system.rootContext=/services
```

Comment:
This property is not needed.  It is defined by CXF with the system property org.apache.cxf.servlet.context.